### ISSUE TITLE
Add SQL template for Python v2

### ISSUE THREAD

**Issue by lucyzhang929** [Azure/azure-functions-sql-extension#1063]
Add template here: https://github.com/Azure/azure-functions-templates/tree/dev/Functions.Templates/Templates-v2

**Comment by LouisDeconinck** [comment]
Is it possible to add multiple rows to the database in one go?

**Comment by lucyzhang929** [comment]
Yes, in Python you can use SqlRowList to add multiple rows. See this sample: https://github.com/Azure/azure-functions-sql-extension/blob/3a5971284fc6bb6dc0d16e4d10f159220de4e9a2/samples/samples-python/AddProductsArray/__init__.py#L16

**Comment by LouisDeconinck** [comment]
Thank you @lucyzhang929 that works perfectly!

I'm building a web scraper using Azure Functions and so now I'm scraping multiple pages and adding information about each page as a JSON object. When I'm done scraping I load all the rows in the database (about 8,000 rows). The whole process takes about 10 minutes. This worked all flawlessly, however I still got a non-breaking error message halfway through the scraping job:
```
Timeout value of 00:30:00 exceeded by function 'Functions.AddRow' (Id: 'aa65943e-8fa1-4f61-9616-47ae138664bd'). Initiating cancellation.
An unhandled host error has occurred.
Microsoft.Azure.WebJobs.Script.WebHost: The host is disposed and cannot be used. Disposed object: 'Microsoft.Azure.WebJobs.Script.WebHost.ScriptLoggerFactory'. Microsoft.Extensions.Logging: Cannot access a disposed object.
Object name: 'LoggerFactory'.
```

Like I mentioned the function didn't stop and all data was successfully stored in the Azure SQL Database so no complaints, but still found it strange I got this message. Is this something to worry about? If so, how to remediate?

**Comment by Charles-Gagnon** [comment]
@LouisDeconinck Can you submit a separate issue for this? That'll make it easier to track and for others to find in the future if they run into similar issues!

**Comment by lucyzhang929** [comment]
@LouisDeconinck, this seems like a functions error, not specific to SQL Bindings. Could you file an issue in the [Azure Functions Host repository](https://github.com/Azure/azure-functions-host/issues)?

**Comment by LouisDeconinck** [comment]
@lucyzhang929 I'm running into another issue. Getting the following error:
`Database on server is not currently available. Please retry the connection later. If the problem persists, contact customer support, and provide them the session tracing ID of '{C16638E0-26B4-4AB7-B831-19F334051195}'.`

Is there anyway how I can detect that the database will not be available and retry writing another time?

I have the following code, but this doesn't work as I believe that the .set() happens asynchronously after the function has already ended.
```python
        max_retries = 10
        retries = 0

        while retries < max_retries:
            try:
                r.set(rows_sql)
                
                total_time = time.time() - start_time

                logging.info(f"End of scraping. Scraped {len(rows)} groups in {total_time}.")
                break

            except Exception as e:
                logging.error(f"An error occurred: {str(e)}")
                retries += 1
                if retries < max_retries:
                    logging.info(f"Retrying after 30 seconds... (retry {retries}/{max_retries})")
                    time.sleep(30)
                else:
                    logging.error(f"An error occurred: {str(e)}")
```

**Comment by lucyzhang929** [comment]
@LouisDeconinck Can you create a separate issue for your question please? https://github.com/Azure/azure-functions-sql-extension/issues/new/choose Thank you!
